### ISSUE TITLE
Minehut Simple Voice Chat

### ISSUE THREAD

**Issue by Thetriplecross** [Minehut/Meta#952]
Is there any way that the mod Simple voice chat could be integrated with minehut? I would like to be able to have a feature like this without having to pay to buy a voice chat plugin in the store. I believe that the DDos protection is what keeps the mod from working. Is there any way that an exception could be made for this specific mod?

### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

**Describe the feature**

**How the feature is useful**

**Comment by Tringlle** [comment]
No, it needs its own port. Its not DDoS prot canceling it. Minehut has stated they will add ability for custom ports to be allocated to a server soon, but until then, not possible.

**Comment by Thetriplecross** [comment]
Ok cool. I saw a lot of people on reddit and discord asking if this was available for minehut. Glad that is on the to-do list for the near-ish future

**Comment by CoreyShupe** [comment]
> Ok cool. I saw a lot of people on reddit and discord asking if this was available for minehut. Glad that is on the to-do list for the near-ish future

There have been no claims of this by any official sources. It is not planned for the near future. Nor is it planned for non-proxy networks or external -> internal comms.
